sympy/logic/boolalg.py

```diff
@@ -2761,7 +2761,7 @@ def simplify_logic(expr, form=None, deep=True, force=False, dontcare=None):
     Parameters
     ==========
 
-    expr : Boolean
+    expr : Boolean expression
```

Comment:
```suggestion
    expr : Boolean
```
I'd let it correspond to sympy's polymorphic Boolean class